contracts/discrepancies/nonce/InternalCallee.sol

```diff
@@ -1,28 +1,50 @@
 // SPDX-License-Identifier: Apache-2.0
 pragma solidity ^0.8.0;
 
-contract Sample {}
+contract Sample {
+  function selfdestruct() external {
```

Comment:
this will throw
```bash
Warning: This declaration shadows a builtin symbol.
```

Change it like below to avoid the warning
```suggestion
  function selfDestruct() external {
```